JASP-Desktop/gui/preferencesmodel.cpp

```diff
@@ -33,6 +34,13 @@ PreferencesModel::PreferencesModel(QObject *parent) :
 	connect(this,					&PreferencesModel::disableAnimationsChanged,	this, &PreferencesModel::animationsOnChanged			);
 
 	connect(LanguageModel::lang(),	&LanguageModel::currentIndexChanged,			this, &PreferencesModel::languageCodeChanged);
+
+	connect(this,					&PreferencesModel::useDefaultInterfaceFontChanged,	[&](){	emit realInterfaceFontChanged(realInterfaceFont()); } );
```

Comment:
Are you aware you can connect signals to signals? Would be prettier than this.
Also, I don't think it is necessary to send the font with it?